Gemfile.lock

```diff
@@ -341,7 +341,7 @@ GEM
       thor (>= 0.18.1, < 2.0)
     rainbow (2.2.2)
       rake
-    rake (12.0.0)
+    rake (11.3.0)
```

Comment:
Thanks for reviewing. Yes, it was intentional.
I should did write in description...